src/vanilla/Templates/Rest/Client/MethodGroupInterfaceTemplate.cshtml

```diff
@@ -19,6 +19,7 @@ namespace @Settings.Namespace
     {
     @foreach(MethodCs method in Model.Methods)
     {
+        if (method.ExcludeFromInterface) continue;
```

Comment:
Quick nit: can you change the use of if to use the `{ }`  so that it fits in our style? (and the other uses)
```  csharp
if (method.ExcludeFromInterface) 
{ 
   continue;
}
```

Reply:
Done.